app/src/main/java/com/example/streammoviesapplication/data/db/TrendingMoviesDao.kt

```diff
@@ -14,7 +15,7 @@ interface TrendingMoviesDao {
     suspend fun insertTrendingMovies(movies: List<TrendingMoviesEntity>)
```

Comment:
you didnt add  @Insert(onConflict = OnConflictStrategy.REPLACE)

Reply:
i did o